bridge/engine/EngineMixer.cpp

```diff
@@ -1527,12 +1527,17 @@ bool EngineMixer::onSctpConnectionRequest(transport::RtcTransport* sender, uint1
 
 void EngineMixer::onSctpEstablished(transport::RtcTransport* sender)
 {
-    logger::debug("SCTP association established", sender->getLoggableId().c_str());
+    logger::info("SCTP association established %s", _loggableId.c_str(), sender->getLoggableId().c_str());
 
     auto barbellIt = _engineBarbells.find(sender->getEndpointIdHash());
-    if (barbellIt != _engineBarbells.cend() && sender->isDtlsClient())
+    if (barbellIt != _engineBarbells.cend())
     {
+        logger::debug("opening barbell webrtc data channel on %s",
```

Comment:
We have to open webrtc data channel from obth sides simultaneously as we want to send the BB msg right away. 
Option would be to wait for the webrtc data channel to be established from the DTLS client side.